### DISCUSSION TITLE
Can't acces GUI on PC

### DISCUSSION THREAD

**Discussion by DannyWagensveld** [theyosh/TerrariumPI#920] [Q&A] [answered]
Herewith another question, my apologies.

I've had TerrariumPI running for a week now and until today I've gotten quite a few things working. A number of sensors and the Tapo 100. My experience with the Raspberry is limited, in the past I did a lot with HomeAssistant but that has little to do with the Raspberry itself.

I've gone through the FAQ but that doesn't get me anywhere.

While I was busy today, the connection in the GUI got disconnected. The browser now keeps indicating that the website is unreachable. I am trying to access the PI from my windows 11 laptop connected to the same wifi network.

I can log into TerrariumPI though by connecting a keyboard and monitor to the PI. I can also log into TerrariumPI via SSH.

I had given the PI a fixed IP address in the DHCP settings of my Ziggo router. I see that the IP address has remained the same, this is also the IP address I use to connect via PuTTY. 

Anyone have an idea in which direction I can look?

I also enabled port forwarding but I don't think that has anything to do with it.

Thanks in advance.

**Comment by theyosh** [comment]
There should be a log file in the folder log in the TerrariumPI folder. If you can upload the logfile here, I can have a look. You can use https://winscp.net/eng/index.php or FileZilla

**Reply by DannyWagensveld** [reply]
I think i should use the commands below? Cause it's not working..

~ $ sudo service terrariumpi stop
~ $ cd /home/pi/TerrariumPI/
-bash: cd: /home/pi/TerrariumPI/: No such file or directory

**Reply by theyosh** [reply]
> I think i should use the commands below? Cause it's not working..
> 
> ~ $ sudo service terrariumpi stop ~ $ cd /home/pi/TerrariumPI/ -bash: cd: /home/pi/TerrariumPI/: No such file or directory

Are you running with docker or manual installation? What kind of Raspberry PI are you using?

**Comment by DannyWagensveld** [comment]
It was manual on a 3B, but i installed TerrariumPI again and now it's working again..

**Reply by DannyWagensveld** [reply]
Could use it for a hour or something, added al the sensors again en after restarting i have the same problem again. TerrariumPI is up and running, can reach it with SSH but can't reach the GUI.

~ $ sudo service terrariumpi stop
~ $ cd /home/pi/TerrariumPI/
-bash: cd: /home/pi/TerrariumPI/: No such file or directory

**Reply by theyosh** [reply]
What happens when you run: `cat /etc/systemd/system/terrariumpi.service`
It should produce some output. Please paste it here

**Reply by DannyWagensveld** [reply]
Unfortunately,  I also get nothing in return

Danny@paludarium:~ $ cat/etc/systemd/system/terrariumpi.service
-bash: cat/etc/systemd/system/terrariumpi.service: No such file or directory

Edit: do have to say that both times I was trying to get the door sensor connected. Maybe I was doing something wrong. Can try to install it again and leave the door sensor for now.

**Reply by theyosh** [reply]
This is getting strange. Which sensors, relays etc did you add?

**Reply by DannyWagensveld** [reply]
A 1 wire temp sensor, bme 280 temp and humidity sensor, tapo 100 smart socket and was trying to get the door sensor to work. All sensor are listed on your website. Tried a relay a couple of days ago but couldn’t get it to work, could switch it with ssh when changing gpio input/output. But then switched to the tapo.

All sensors were working for a couple of days. 

Have to say a lot of what i do is try and error, I do read tutorials but it’s quite difficult to find the right info.

**Reply by theyosh** [reply, marked as answer]
Ok, what you can try is: Reinstall the TerrariumPI stoftware, but do not reboot when done. That is only needed a first time.
Than start it in debug mode: https://theyosh.github.io/TerrariumPI/faq/how-to-debug-terrariumpi/ (pages are currently looking a bit strange...)

And than add the sensors and relays.. This should produce output when something goes wrong. But it will stop when you press Ctrl+C

Also, maybe a wrong GPIO pin is used: https://pinout.xyz/ Some GPIO pins are reserved, and should not be used.

But I have no clue what is happening...

**Reply by DannyWagensveld** [reply]
I will do today or tomorrow and let you know the outcome. Thanks for your help and great software.

**Reply by theyosh** [reply]
Any progress?? I think it is strange....

**Reply by DannyWagensveld** [reply]
I reinstalled TerrariumPi and it’s working since then, also my sensors are working. So solved for now and will let you know if something strange happens
